dask/dataframe/tests/test_rolling.py

```diff
@@ -111,7 +109,9 @@ def test_rolling_funtions_dataframe():
 ])
 @pytest.mark.parametrize('window', [1, 2, 4, 5])
 @pytest.mark.parametrize('center', [True, False])
-@pytest.mark.parametrize('axis', [0, 1, 'rows', 'columns'])
+@pytest.mark.parametrize('axis', [0, 'columns'])
+@pytest.mark.skipif(LooseVersion(pd.__version__) < '0.18.0',
+                    reason="rolling object not supported")
```

Comment:
Thanks for letting me know. I saw there was a merge conflict so I rebased the changes and have updated to drop all of these. (Will push after a resolve one issue.)